doc/source/api/index.rst

```diff
@@ -83,8 +83,8 @@ API
    .. toctree::
        :maxdepth: 1
 
-       python
-       python_api_ref
```

Comment:
Removed `python_api_ref` as it wasn't entirely necessary and was causing issues for some reason.